spec/chewy/type/import_spec.rb

```diff
@@ -43,6 +43,45 @@
       end
     end
 
+    describe 'using scopes with includes' do
+      if const_defined?('::ActiveRecord') && ::ActiveRecord::VERSION::MAJOR > 4
```

Comment:
Doesn't work because it wont construct the right (joined) query.

```ruby
Country.includes(:cities).where("cities.id IS NOT NULL").to_sql
=> "SELECT \"countries\".* FROM \"countries\" WHERE (cities.id IS NOT NULL)"
# will throw column not found for 'cities.id'
```

vs.

```ruby
Country.includes(:cities).where.not(cities: { id: nil }).to_sql
=> "SELECT \"countries\".\"id\" AS t0_r0, \"countries\".\"name\" AS t0_r1, \"countries\".\"country_code\" AS t0_r2, \"countries\".\"rating\" AS t0_r3, \"cities\".\"id\" AS t1_r0, \"cities\".\"country_id\" AS t1_r1, \"cities\".\"name\" AS t1_r2, \"cities\".\"rating\" AS t1_r3 FROM \"countries\" LEFT OUTER JOIN \"cities\" ON \"cities\".\"country_id\" = \"countries\".\"id\" WHERE (\"cities\".\"id\" IS NOT NULL)"
```
```